dbt/adapters/databricks/column.py

```diff
@@ -1,9 +1,22 @@
 from dataclasses import dataclass
+from typing import ClassVar, Dict
 
 from dbt.adapters.spark.column import SparkColumn
 
 
 @dataclass
 class DatabricksColumn(SparkColumn):
+    TYPE_LABELS: ClassVar[Dict[str, str]] = {
+        "LONG": "BIGINT",
```

Comment:
Why do we need this?

Reply:
This is a port of https://github.com/dbt-labs/dbt-spark/pull/358.